app/Modules/Currencies/Support/CurrencyConverterFactory.php

```diff
@@ -18,7 +18,15 @@ class CurrencyConverterFactory
 {
     public static function create()
     {
-        $class = 'IP\Modules\Currencies\Support\Drivers\\' . config('fi.currencyConversionDriver');
+
+$test = config('fi.currencyConversionDriver');

```

Comment:
Please properly format the code, may also rename `$test` to `$fallback_driver` which is more clear

Reply:
Resolved in fa16c07